### PR TITLE
Home page skeleton

### PR DESCRIPTION
**Pull request by mattgfisch**
Trello Story: https://trello.com/c/c3LkVUp8

Merging this branch to development will add a home page skeleton that we can work off of. Here is a screenshot of what it looks like:

screen shot 2017-07-09 at 11 11 35 pm

To accomplish this I:
- Removed the unneeded React component
- Created a page header and a jumbotron (Bootstrap)
- Added custom styling to buttons
- Established a color-scheme

Caveats:
- The hook is just a stub
- Although it was specified in the Acceptance Criteria, there is no logo

Although it would be nice to have the final content for our hook and very nice to have a logo, I think it's a bit early to focus on those things. I'll add creating a logo and polishing the main page content to the tasks we'll complete in the final week before Alpha testing.

### CODE REVIEW

File: src/client/views/home.hbs [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+  <p> 
+    This is example text that isn't actually our site hook, but we'll work on that. If you want to learn more, join a session.
+  </p>
+  <p><label class='btn btn-lg home-page-btn' href='#' type='button'>Join Session</label></p>   
+</div>
 
-<div>{{> ex}}</div>
-
-<div id='root'></div>
 <script src='{{assetPrefix}}/js/hello.js'></script>
```
Comment (mpace965) [inline src/client/views/home.hbs]: Maybe get rid of this script tag too.

**Review by mpace965 — CHANGES_REQUESTED** [review]
I think the homepage looks really good!

It would also be prudent though to override the bootstrap `$brand` colors like we talked about. [Here](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L7) is a complete listing of them. This would allow you to get rid of some (all?) of the `!important` rules, and will also make further styling easier and more consistent.

Doing this essentially entails redefining the sass variables after you import bootstrap. [This forum post](https://laracasts.com/discuss/channels/elixir/how-to-modify-things-inside-bootstrap-sass/replies/109552)  describes a good method I think.

So, to clarify, I don't think any of the styling needs to change, but try out the variable overriding and see if that cuts down on having to explicitly define a bunch of colors.

File: src/client/views/home.hbs (hunk 1 above)
Comment (mattgfisch) [reply src/client/views/home.hbs]: Sounds good! That would be a much cleaner solution and, like you said, would make it easy to implement the color scheme in the future. I'll get rid of the script tag as well and then repush!

File: src/client/assets/styling/main.scss [hunk 2]
```diff
@@ -1,2 +1,48 @@
-$icon-font-path: "~bootstrap-sass/assets/fonts/bootstrap/";
-@import '~bootstrap-sass/assets/stylesheets/_bootstrap.scss'
\ No newline at end of file
+$icon-font-path: '~bootstrap-sass/assets/fonts/bootstrap/';
+@import 'variables';
+@import '~bootstrap-sass/assets/stylesheets/_bootstrap.scss';
+@import 'overwrites';
+
```
Comment (mattgfisch) [inline src/client/assets/styling/main.scss:5]: I brought in our own variables to take the place of some Bootstrap variables and then used our own variables in areas of our home page where bootstrap doesn't automatically provide styling. I also bring in `overwrites` to overwrite Bootstrap behavior that doesn't rely on variables.

File: src/client/assets/styling/main.scss [hunk 3]
```diff
@@ -1,2 +1,48 @@
...
+}
+
+.home-page-btn {
+  background-color: $button-color;
+  color: $canvas-color;
+  &:hover, &:focus {
+    background-color: $button-color-hover; 
+  }
+}
+
```
Comment (mattgfisch) [inline src/client/assets/styling/main.scss:46]: Ultimately some of this content will be refactored to a homepage specific stylesheet, but I'll wait until I work on the Sessions page to do that. That way, I'll know which styles are overlapping and which are specific to each page.

**Review by mpace965 — APPROVED** [review]
Nice, code and site looks very clean!! :100:

**Comment by mpace965** [comment]
Don't forget to squash (like I did - :laughing:)

**Pull request merged**